ISTQB-CTEL-TM-OTM-Guide.md

```
@@ -0,0 +1,1522 @@
+# 🏆 ISTQB® Certified Tester Expert Level – Test Management
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)